vscode-build/vscode.patch

```diff
@@ -1,35 +1,29 @@
-diff --git a/build/gulpfile.vscode.js b/build/gulpfile.vscode.js
-index 13c20bed989..2a8452a08ab 100644
---- a/build/gulpfile.vscode.js
-+++ b/build/gulpfile.vscode.js
-@@ -35,13 +35,14 @@ const { compileExtensionsBuildTask } = require('./gulpfile.extensions');
+diff --git a/build/gulpfile.compile.js b/build/gulpfile.compile.js
+index c4947e76cbf..e46583b9073 100644
+--- a/build/gulpfile.compile.js
++++ b/build/gulpfile.compile.js
+@@ -22,7 +22,7 @@ function makeCompileBuildTask(disableMangle) {
+ }
+ 
+ // Full compile, including nls and inline sources in sourcemaps, mangling, minification, for build
+-const compileBuildTask = task.define('compile-build', makeCompileBuildTask(false));
++const compileBuildTask = task.define('compile-build', makeCompileBuildTask(true));
```

Comment:
The `true` here is to disable mangling. Minification still happens. For some reason the mangling was causing the build to completely hang on my machine.